python/instrumentation/openinference-instrumentation-llama-index/examples/multimodal_stream_chat.py

```diff
@@ -0,0 +1,64 @@
+from llama_index.core.multi_modal_llms.generic_utils import load_image_urls
+from llama_index.multi_modal_llms.openai import OpenAIMultiModal
+from llama_index.multi_modal_llms.openai.utils import (
+    generate_openai_multi_modal_chat_message,
+)
+from openinference.instrumentation.llama_index import LlamaIndexInstrumentor
+from openinference.instrumentation.openai import OpenAIInstrumentor
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk import trace as trace_sdk
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+
+endpoint = "http://127.0.0.1:6006/v1/traces"
+tracer_provider = trace_sdk.TracerProvider()
+tracer_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint)))
+tracer_provider.add_span_processor(
+    span_processor=SimpleSpanProcessor(span_exporter=ConsoleSpanExporter())
+)
+
+LlamaIndexInstrumentor().instrument(tracer_provider=tracer_provider)
+OpenAIInstrumentor().instrument(tracer_provider=tracer_provider)
```

Comment:
let's keep the examples focused solely on llama index
```suggestion
```

Reply:
Done